mobify/progressive-web-scaffold
# Add Global Styles

Add some of the necessary global styles to the scaffold for use with Merlin's Potions.

## Changes
- Add form styles, such as radio buttons, checkboxes and buttons
- Add some styles for common components, like Icon, List and ListTile
- Tweak some base styles, such as typography
- Add base colors: brand, accent, success, error, neutrals, etc.

## How to test-drive this PR
- Run `npm run styleguide`
- Visit [http://localhost:4000/](http://localhost:4000/)
- Confirm that the above mentioned changes are reflected in the styles
